package.json

```diff
@@ -89,6 +89,7 @@
     "rollup-plugin-terser": "^5.1.1",
     "rollup-plugin-typescript2": "^0.24.0",
     "rxjs": "^6.5.2",
+    "ts-toolbelt": "^3.8.63",
```

Comment:
This isn't a dev dep if it's used in src.

Reply:
I think you're right. I was thinking because all the types and interfaces disappear, it won't need to be a normal dependency, but if the import is still there... yeah.

Reply:
Fixed in 6a76ebb. If it's any consolation, here's the minified bundle size:

<img src="https://badgen.net/bundlephobia/min/ts-toolbelt?label=ts-toolbelt" />